docker-compose.yml

```diff
@@ -43,19 +43,18 @@ services:
     volumes:
       - db_data_ci:/var/lib/postgresql/data
     ports:
-      - ${CI_POSTGRES_PORT}:${LOCAL_POSTGRES_PORT}
+      - ${CI_POSTGRES_PORT}:5432
 
   graphql-engine-ci:
     <<: *graphql_engine
     profiles: ['ci']
     ports:
-      - ${CI_HASURA_PORT}:${CI_HASURA_PORT}
+      - ${CI_HASURA_PORT}:8080
```

Comment:
Should this be 8081?
```suggestion
      - ${CI_HASURA_PORT}:8081
```

Reply:
The second value is the internal port; the first value is the one that's exposed to outside the container. I'm intentionally reverting these to the default because it reduces the amount of configuration needed for the service.

Reply:
Anyway, I found a way to get this all to work that will allow removing those two `-ci` containers entirely.